youtube/__init__.py

```diff
@@ -7,6 +7,10 @@
 yt_app = flask.Flask(__name__)
 yt_app.config['TEMPLATES_AUTO_RELOAD'] = True
 yt_app.url_map.strict_slashes = False
+# yt_app.jinja_env.trim_blocks = True
+# yt_app.jinja_env.lstrip_blocks = True
+
+
```

Comment:
Avoid adding cruft to the codebase. I recommend using `git diff` right before performing each commit and reading/reviewing/pondering all the changes you made to make sure you didn't introduce anything extra or did something wrong or forgot something. I catch mistakes (and bugs) I make very frequently by doing this before committing.

Reply:
This was intentional, so I don't need to search it again should it be needed (in the future).
Sometimes you are not 100% sure that the noisy newlines/whitespaces get ignored correctly (from the browser).